frontend/apps/reader/readerui.lua

```diff
@@ -455,6 +455,19 @@ function ReaderUI:init()
     -- Book information).
     self.doc_settings:saveSetting("doc_props", self.document:getProps())
 
+    -- Set "reading" status if there is no status.
+    local summary = self.doc_settings:readSetting("summary")
+    if not (summary and summary.status) then
+        if not summary then
+            summary = {}
+        end
+        summary.status = "reading"
+        summary.modified = os.date("%Y-%m-%d", os.time())
```

Comment:
https://github.com/koreader/koreader/blob/20ac8004d65089764cad517dcf1f4c44919d2677/frontend/ui/widget/bookstatuswidget.lua#L52-L78
may consider what you filled as "Complete, use it as-is", and it then may miss some fields (rating, node) ?

There are other bits of code (coverbrowser, readerstatus) that also are made to handle an incomplete status (empty string) as reading, so these would now only be needed for "legacy, not yet reopened document", which would need them to have comments updated.
I would in this PR make the minimal setup so these other code works as is - ie. `status=""`, and not creating `modified`, so `modified` would mean it has been explicitely updated by the user in BookStatusWidget - and we don't have to duplicate (here and in BookStatusWidget) the initial values for all these fields.

Reply:
The idea is to set the `status` to "reading" to help in https://github.com/koreader/koreader/issues/9500.
I suppose a user explicitly changes the status by opening a book.
The `modified` field is used to preserve the `status` from resetting to "" by BSW. This does not break any logic of other modules (`rating` and `note` fields can be nil).

To maintain old books without a summary the current code of other modules must be kept.
(Or modified in consistency with this PR, but these changes should be postponed until finding the common agreement about the new status buttons).

Reply:
OK, just wanted to be sure you have thought about all that :)